api/app/wildfire_one/wfwx_api.py

```diff
@@ -379,3 +381,30 @@ async def get_dailies_for_stations_and_date(session: ClientSession,
     yesterday_dailies = await mapper(raw_dailies)
 
     return yesterday_dailies
+
+
+async def get_station_groups(mapper=weather_station_group_mapper):
+    """ Get the station groups created by all users from Wild Fire One internal API. """
+    async with ClientSession() as session:
+        header = await get_auth_header(session)
+        all_station_groups = fetch_paged_response_generator(session,
+                                                            header,
+                                                            BuildQueryStationGroups(),
+                                                            'stationGroups',
+                                                            use_cache=False)
+        # Map list of stations into desired shape
+        mapped_station_groups = await mapper(all_station_groups)
+        logger.debug('total station groups: %d', len(mapped_station_groups))
```

Comment:
I don't think we'll need this logger statement anymore, will we?

Reply:
Yep we might, and we have it elsewhere for other calls.